cli/args/flags.rs

```diff
@@ -458,7 +458,9 @@ impl Flags {
       Some(files.clone())
     } else if let Run(RunFlags { script }) = &self.subcommand {
       if let Ok(module_specifier) = deno_core::resolve_url_or_path(script) {
-        if module_specifier.scheme() == "file" {
+        if module_specifier.scheme() == "file"
+          || module_specifier.scheme() == "npm"
```

Comment:
This was fun - I tried this PR with Vite and couldn't get it to write to a lockfile. Turns out we never discover configuration file is the entrypoint is a `npm` package (eg. `deno run -A --unstable npm:vite`). This conditional now allows to discover configuration file in such case. Should I factor it out to a separate PR?

Reply:
Oh, interesting. I think this makes sense to do.